### DISCUSSION TITLE
How to install the addon after build

### DISCUSSION THREAD

**Discussion by AngelLM** [visose/Robots#261] [Q&A] [answered]
Hello, I want to develop a postprocessor for Fanuc robots. 
I have forked the repository and I have been able to build it without problems as it is. 
I have doubts about how to install the addon once the build is done. How would it be done? 

Thanks!

**Comment by visose** [comment, marked as answer]
Hi, if you build in `Debug` configuration it will place the files under the packages folder for Rhino 7. This means just by building the plugin it will install it, just need to open Rhino 7 + Grasshopper and the plugin will be there.
In `Release` configuration it will create the files under `/artifacts/bin/`. If you just want to manually install the plugin there are multiple ways to do this, but I would recomment to copy the release files to the same packages folder where debug is placing them to avoid clashes in the dev machine.
